rules/windows/process_creation/proc_creation_win_susp_jwt_token_search.yml

```diff
@@ -17,15 +18,9 @@ logsource:
     product: windows
 detection:
     selection:
-        CommandLine|contains:
-            - 'eyJ0eXAiOi' # {"typ":
-            - 'eyJhbGciOi' # {"alg":
-            - ' eyJ0eX'
-            - ' "eyJ0eX"'
-            - " 'eyJ0eX'"
-            - ' eyJhbG'
-            - ' "eyJhbG"'
-            - " 'eyJhbG'"
+        CommandLine|regex:
+            - '(?i)findstr.*(?i)eyJhbG[a-zA-Z0-9]*'
+            - '(?i)findstr.*(?i)eyJ0eX[a-zA-Z0-9]*'
```

Comment:
This will reduce the coverage by a lot and invalidate the original logic of the rule.

`findstr` is only one of many ways one could search for these strings, do not take only public reporting as a use case.

If you say this is creating a lot of FPs. Please provide some anonymized examples. As the presence of JWT in CLI is common but limited and can be excluded.

We could downgrade it to `low` and/or move it as a hunting rule, if necessary and after seeing some data if possible.